File: MODUL 2 PEMROGRAMAN 1/PRAK204-2310817120002-SITIRATNADWINTASARI.c
#include <stdio.h>
#include <math.h>
int main (){
    //input//
    float jari2, tinggi, v, l, k;
    scanf("%f", &jari2);
    scanf("%f", &tinggi);
    v = (22 * pow(jari2,2) * tinggi)/7;
    l = (2 * 22 * jari2 * (jari2 + tinggi))/7;
    k = (2 * 22 * jari2)/7;
    //output//
    printf("Volume = %.2f\n", v);
    printf("Luas = %.2f\n", l);
    printf("Keliling = %.2f\n",k);

    return 0;
}

File: MODUL 2 PEMROGRAMAN 1/PRAK204-2310817120002-SITI RATNA DWINTA SARI.py
import math
#input 1
jari2 = float (input())
tinggi = float (input())
v = 22/7 *(jari2**2) * tinggi
k = 2 * (22/7) * jari2 * (jari2 + tinggi)
l = 2 * (22/7) *jari2
print(f"Volume = {v:.2f}")
print(f"Luas = {k:.2f}")
print(f"Keliling = {l:.2f}")

#input 2
jr2,tinggi1 = map(float, input().split())
v1 = 22/7 * (jr2**2) * tinggi1
k1 = 2 * (22/7) * jr2 * (jr2+ tinggi1)
l1 = 2 * (22/7) *jr2
print(f"Volume = {v1:.2f}")
print(f"Luas = {k1:.2f}")
print(f"Keliling = {l1:.2f}")